src/three/plugins/ImplicitTilingPlugin.js

```diff
@@ -0,0 +1,56 @@
+import {SUBTREELoader} from "./SUBTREELoader.js";
+
+export class ImplicitTilingPlugin {
+
+	constructor() {
+		this.name = 'IMPLICIT_TILING_PLUGIN';
+	}
+
+	init(tiles) {
+
+		this.tiles = tiles;
+
+	}
+
+	preprocessNode(tile, uri) {
+		if (tile.implicitTiling) {	//only for root
+
+			// Store the infos from the tileset
+			tile.__availableLevels = tile.implicitTiling.availableLevels;
+			tile.__subdivisionScheme = tile.implicitTiling.subdivisionScheme;
+			tile.__subtreeLevels = tile.implicitTiling.subtreeLevels;
+			tile.__subtrees = tile.implicitTiling.subtrees;
+			tile.__subtreeDivider = tile.__subdivisionScheme === "QUADTREE" ? 4 : 8;
+			tile.__subtreeUri = tile.implicitTiling.subtrees.uri;
+			tile.__contentUri = uri ?? tile.content?.uri;
```

Comment:
What's the reason for duplicating this information? It looks like it's already available on the tile in the implicitTiling object - why not use those fields?

Reply:
It was to avoid the longer call `tile.implicitTiling.XXX`.
I'll remove some. The ones I keep seem to be relevant but could also be removed if you deem it necessary